webAO/client.js

```diff
@@ -1111,6 +1129,7 @@ class Client extends EventEmitter {
       const trackname = safeTags(args[i]);
       const trackindex = i - 1;
       document.getElementById('client_loadingtext').innerHTML = `Loading Music ${i}/${this.music_list_length}`;
+      document.getElementById('client_loadingbar').value = this.char_list_length + this.evidence_list_length + i;
```

Comment:
because the music is all in one packet it immediatly snaps from 0 to to whatever amount of tracks the server has.
it would update if you single step the js :^)

Reply:
the loading bar is the total progress including the music